src/styles/index.scss

```diff
@@ -1,3 +1,11 @@
+@import './utils/variables';
+@import './utils/mixins';
+@import 'blocks/card';
+@import 'blocks/stars';
+
 body {
   margin: 0;
+  padding: 50px;
+  background-color: #f0f0f0;
```

Comment:
Make sure to use variables for colors to improve maintainability of the code. Refer to the 'variables.scss' file that you've imported to define and use color variables.